conf/config.level0.neon

```diff
@@ -16,6 +16,9 @@ conditionalTags:
 		phpstan.rules.rule: %featureToggles.newStaticInAbstractClassStaticMethod%
 
 services:
+	-
+		class: PHPStan\Rules\Classes\ClassConstantRule
```

Comment:
Why is this added here? It's not needed at all, the rule has `#[RegisteredRule(level: 0)]` attribute.